src/commands/Tools/Websearch/igdb.ts

```diff
@@ -144,4 +143,24 @@ export default class extends RichDisplayCommand {
 		if (!platforms || isArrayOfNumbers(platforms)) return fallback;
 		return platforms.map((platform) => platform.name).join(', ');
 	}
+
+	private buildBody(game: string) {
+		return [
+			[`search "${game}"`, 'limit 10', 'offset 0;'].join('; '),
```

Comment:
Any specific reason why we need to make an array for this?

Reply:
Resolved in my review. As for this versus the old way of a long raw string, that was a decision by me. It just makes it more readable. Anyway with my suggestion it only has 2 array joins. One of the entire body which joins on `'; '` and one for the fields which join on `','`